{"level":"info","memory_mb":13.434288024902344,"message":"Phase started: Extract","phase":"Extract","timestamp":65}
{"level":"error","message":"Error occurred relation \"users\" does not exist","stack":"error: relation \"users\" does not exist\n    at C:\\Users\\arnav\\OneDrive\\Desktop\\PROGRAMMING\\project web development\\ETL\\elt-khund-pipeline-benchmark\\node_modules\\.pnpm\\pg-pool@3.10.1_pg@8.16.3\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Case1Sequential.execute (C:\\Users\\arnav\\OneDrive\\Desktop\\PROGRAMMING\\project web development\\ETL\\elt-khund-pipeline-benchmark\\src\\cases\\case1-sequential.ts:29:27)\n    at async main (C:\\Users\\arnav\\OneDrive\\Desktop\\PROGRAMMING\\project web development\\ETL\\elt-khund-pipeline-benchmark\\src\\index.ts:30:23)","timestamp":99}
